First pass at static site

mattleff opened the pull request.

mattleff commented:
@bountonw This isn't actually ready for review. I just need to test the GitHub action...as you were.

bountonw reviewed (approved):
Approving by faith so that it is ready for you when you need it.

mattleff commented:
@bountonw There's still a couple TODOs before we merge this, but the deployment is working now: https://bountonw.github.io/translate/

We need to redo the [home/introduction page](https://bountonw.github.io/translate/). You can give me whatever you want for there. Also, what should the title (upper left corner) be? Can you give me the copyright line (or whatever you want to say there) in Thai? Let me know if there are any other changes that we need for v1 (or v0.1).

mattleff commented:
@bountonw This is ready for a review: https://bountonw.github.io/translate/

I've added a content last updated date and added book links to the home page. Let me know what else you want to change before we merge this in.

bountonw commented:
Note to self:

TODO:

- [ ] Site name
- [ ] Hello World replacement
- [ ] Introductory paragraph

bountonw commented:
@mattleff Instead of "Hello world" Can we do "welcome...."? Would it be possible to do this in 3 languages (defaulting to Thai, but with clickable flags to change the language)?

English: Welcome to Saeng Haeng Khwam Jing
Thai: ยินดีต้อนรับเข้าสู่เว็บไซต์แสงแห่งความจริง
Lao: ຍິນດີຕ້ອນຮັບສູ່ເວັບໄຊຕ໌ແສງແຫ່ງຄວາມຈິງ
